FR #966: Add recency to metadata (#1374)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/metadata/recency/dataset_instance_complete.md

# Dataset Instance Complete

Dataset Instance Complete provides a boolean value to indicate that the dataset instance is considered complete by the DataGenerator.  The definition of complete is determined by the DataGenerator and should be provided in documentation provided by the DataGenerator. For Datasets that are time series, the Complete value indicates that the time sector is complete and therefore is located in as key value in a time sector. For datasets that are not time series, the Complete value indicates that the dataset is complete and therefore is a property of the recency object.

> The Dataset Instance Complete MUST be present if the recency metadata the Dataset Instance is not an over-time dataset. For over-time datasets the field is optional. The DatasetInstanceComplete MUST be of type Boolean.

Reviewer: ```suggestion
The Dataset Instance Complete MUST be present if the recency metadata the Dataset Instance is not a time-series dataset. For time-series datasets the field is optional. The DatasetInstanceComplete MUST be of type Boolean.
```

Reviewer: I expect time-series (and non-time-series) are more easily understood terms vs. over-time or not over-time but could just be a stylistic preference. Incorporate if you agree or resolve if not (would require an adjustment in a few areas if you agree).
